ui/lib/kv/addon/routes/list-directory.js

```diff
@@ -34,6 +34,9 @@ export default class KvSecretsListRoute extends Route {
           : models;
       })
       .catch((err) => {
+        if (err.httpStatus === 403) {
+          return 403;
+        }
         if (err.httpStatus === 404) {
           this.has404 = true;
```

Comment:
Just out of curiosity - where is `this.has404` being used?

Reply:
Good question. It will eventually be used, but not yet so I'll remove the this.has404.